README.md

```diff
@@ -33,6 +33,7 @@ build.yaml
     - ansible.content_builder.run
 ```
 MANIFEST.yaml
+### For Network collection scaffolding
```

Comment:
This is not only for network collection scaffolding.

Reply:
Can we also add an entry for type: module_openapi?

Reply:
added